website/content/docs/ecs/architecture.mdx

```diff
@@ -62,56 +76,103 @@ This diagram shows an example timeline of a task shutting down:
 - **T0**: ECS sends a TERM signal to all containers. Each container reacts to the TERM signal:
   - `consul-client` begins to gracefully leave the Consul cluster.
   - `health-sync` stops syncing health status from ECS into Consul checks.
-  - `sidecar-proxy` ignores the TERM signal and continues running until the `user-app` container exits. This allows the application container to continue making outgoing requests through the proxy to the mesh.
-  - `user-app` exits if it is not configured to ignore the TERM signal. The `user-app` container will continue running if it is configured to ignore the TERM signal.
+  - `sidecar-proxy` ignores the TERM signal and continues running until the `user-app` container
+    exits. The custom entrypoint command, `consul-ecs envoy-entrypoint`, monitors the local ECS task
+    metadata. It waits until the `user-app` container has exited before terminating Envoy. This
+    enables the application to continue making outgoing requests through the proxy to the mesh for
+    graceful shutdown.
+  - `user-app` exits if it is not configured to ignore the TERM signal. The `user-app` container
+    will continue running if it is configured to ignore the TERM signal.
 - **T1**:
-  - `health-sync` updates its Consul checks to critical status and exits. This ensures this service instance is marked unhealthy.
+  - `health-sync` does the following:
+    - It updates its Consul checks to critical status and exits. This ensures this service instance is marked unhealthy.
+    - If ACLs are enabled, it runs `consul logout` for the two tokens created by the `consul-client` and `mesh-init` containers.
+      This removes those tokens from Consul. If `consul logout` fails for some reason, the ACL controller will remove the tokens
+      after the task has finished stopped.
```

Comment:
Is "finished stopped" the intended wording? I know that `STOPPED` is a ECS task state, but in this context the wording feels a bit awkward.